gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/pom.xml

```diff
@@ -32,7 +32,7 @@
 
     <properties>
         <jetty94.wiremock.version>9.4.44.v20210927</jetty94.wiremock.version>
-        <gravitee-connector-http.version>1.1.1-SNAPSHOT.6967-remove-accept-encoding-client-header</gravitee-connector-http.version>
+        <gravitee-connector-http.version>1.1.1-SNAPSHOT</gravitee-connector-http.version>
```

Comment:
⚠️ Will not be replaced in the release process because of this:
https://github.com/gravitee-io/gravitee-api-management/blob/3.15.x/.circleci/config.yml#L795

Since gravitee-connector-http uses now semantic release (at least for the master branch), we added this line to prevent weird behaviour.

So for your case, I would advise remove the SNAPSHOT suffix. (And yes it implies to release the plugin first)

Reply:
Ok it's only for test purpose. You can ignore my previous comment